src/transaction/log_checkpoint_info.cpp

```diff
@@ -167,7 +168,7 @@ namespace cublog
       }
 
     size += serializator.get_packed_bigint_size (start_offset + size);
-    for (const checkpoint_sysop_info sysop_info : m_sysops)
+    for (const auto &sysop_info : m_sysops)
```

Comment:
seems like there are more copies being done:
~295
```
for (auto chkpt : m_trans)
```
~344
```
for (auto sysop : m_sysops)
```
~394
```
for (auto chkpt : m_trans)
```
or am I mistaken?

Reply:
No, you're right. I'll push a fix.